sections/custom-fields.md

```diff
@@ -124,8 +124,8 @@ POST /api/v1/users/<user_id>/custom_field_values
 
 ### Updating Custom Field Values
 ```
-PUT /api/v1/projects/<project_id>/custom_field_values
-PUT /api/v1/users/<user_id>/custom_field_values
+PUT /api/v1/projects/<project_id>/custom_field_values/<custom_field_value_id>
+PUT /api/v1/users/<user_id>/custom_field_values/<custom_field_value_id>
```

Comment:
convention here is to have the last id in the URL to just read id. Otherwise LGTM

    PUT /api/v1/users/<user_id>/custom_field_values/<id>